konduit-serving-vertx-protocols/konduit-serving-http/src/main/java/ai/konduit/serving/vertx/protocols/http/verticle/InferenceVerticleHttp.java

```diff
@@ -149,7 +153,7 @@ private HttpServerOptions createOptions(int port, boolean useSsl, String sslKeyP
                 .setCompressionSupported(true)
                 .setTcpKeepAlive(true)
                 .setTcpNoDelay(true)
-                .setAlpnVersions(Arrays.asList(HttpVersion.HTTP_1_0,HttpVersion.HTTP_1_1))
+                .setAlpnVersions(Arrays.asList(HttpVersion.HTTP_1_0, HttpVersion.HTTP_1_1))
```

Comment:
A lot of browsers are going to force http2 soon. This might not be such a good idea.